ignite/services/network/networkchain/prepare.go

```diff
@@ -253,6 +257,24 @@ func (c Chain) applyGenesisValidators(ctx context.Context, genesisVals []network
 	return c.updateConfigFromGenesisValidators(genesisVals)
 }
 
+// applyParamChanges applies the param changes into the genesis
+func applyParamChanges(
+	genesis *cosmosgenesis.Genesis,
+	paramChanges []networktypes.ParamChange,
+) error {
+	for _, pc := range paramChanges {
+		if err := genesis.Update(
+			jsonfile.WithKeyValueByte(cosmosgenesis.ParamField(pc.Module, pc.Param), pc.Value),
+		); err != nil {
+			return fmt.Errorf("failed to apply param change to genesis: module: %v, param: %v, value: %v: %w",
+				pc.Module, pc.Param, pc.Value, err,
+			)
+		}
+	}
```

Comment:
This can also reduce the `genesis.Update` calls:

```suggestion
	changes := make([]jsonfile.UpdateFileOption, len(paramChanges))
	for i, pc := range paramChanges {
		changes[i] = jsonfile.WithKeyValueByte(cosmosgenesis.ParamField(pc.Module, pc.Param), pc.Value)

	}
	if err := genesis.Update(changes...); err != nil {
		return fmt.Errorf("failed to apply param change to genesis: %w", err)
	}
```